packages/wine.rb

```diff
@@ -14,9 +14,17 @@ class Wine < Package
   })
 
   depends_on 'bison'
-  depends_on 'sommelier'
   depends_on 'flex'
   depends_on 'freetype'
+  depends_on 'ncurses'
+  depends_on 'cups'
+  depends_on 'gettext'
+  depends_on 'libgudev'
+  depends_on 'mesa'
```

Comment:
I do know mesa comes along for the ride with sommelier so we don't need it here.